src/pages/schemas/content.tsx

```diff
@@ -51,7 +51,7 @@ export default function SchemaContent() {
             await dispatch(postColumnTagsAndDescriptionInfoAction(selectedUid, tags, description));
         }
         showSuccessToast("saved successfully");
-        dispatch(fetchDataBaseInfoAction());
+        // dispatch(fetchDataBaseInfoAction());
```

Comment:
Again make sure no commented lines should be included for merging in main branch